frontend/src/main/components/Commons/ManageCows.js

```diff
@@ -2,14 +2,16 @@ import React from "react";
 import { Card, Button, Row, Col } from "react-bootstrap";
 import cowHead from "./../../../assets/CowHead.png"; 
 
-// add parameters 
-const ManageCows = ({userCommons,commons, onBuy, onSell}) =>  {
+
+// add parameters
+const ManageCows = ({userCommons, commons, onBuy, onSell}) =>  {
     // update cowPrice from fixture
+    // Stryker disable ArithmeticOperator : line 24: not very important and unsure of how to test this
     return (
         <Card>
         <Card.Header as="h5">Manage Cows</Card.Header>
         <Card.Body>
-            {/* change $10 to info from fixture */}
+            {/* why does removing the optional chaining question mark turn the entire page white? */}
```

Comment:
Comments such as these don't belong in code that's going to be merged into main.

If you have this question let's get it answered, but then let's remove this comment.